sentry_sdk/utils.py

```diff
@@ -197,15 +197,17 @@ def __init__(
         self.version = version
         self.client = client
 
-    @property
-    def store_api_url(self):
```

Comment:
How about we keep `store_api_url` in there not to break anybody who may depend on it? (mark it as deprecated if we have a way of doing so)

Reply:
This is not really supposed to be public API. The public API is documented at https://getsentry.github.io/sentry-python/api.html

We need to do a better job with naming conventions (leading underscores) though.

Reply:
We'd be unhappy if some web framework did a similar change. Even if this had the underscore, there's nothing stopping us keeping this around for being nice to whomever depends on this.

But since I don't know what has been the history of maintaining compatibility in the SDK, I do not make a strong call for this.

Reply:
I took the liberty to add this back and added tests to cover the new `get_api_url`.